### PR TITLE
Documentation of how to use 1.3 modules in CoffeeScript

### PR DESCRIPTION
**Pull request by GeoffreyBooth**
Per [discussion](https://github.com/meteor/meteor/issues/5788#issuecomment-181988193) with @avital, some documentation of how to use CommonJS to work with modules in CoffeeScript in Meteor 1.3.

### CODE REVIEW

**Comment by tmeasday** [comment]
@benjamn I think perhaps you should look over this?

**Comment by zol** [comment]
Hey, Thanks for your PR. Please sign our [contributor agreement](https://contribute.meteor.com/) before we can accept it.

**Comment by GeoffreyBooth** [comment]
Done!

**Comment by zol** [comment]
Thanks!

**Comment by Dyscrete** [comment]
CoffeeScript hasn't had a lot of progress lately, I think the switch to ES6/7 should be an encouraged standard.

**Comment by GeoffreyBooth** [comment]
@Dyscrete CoffeeScript doesn’t really need a lot of progress, since there’s very little in ES2015 that CoffeeScript didn’t already have. Modules are the only substantial difference, in that CoffeeScript (like ES5) relied on CommonJS. CommonJS `require` statements [are required in Meteor 1.3 for certain situations](http://guide.meteor.com/v1.3/structure.html#using-require), even when using ES2015, so CommonJS support will be a part of Meteor for the foreseeable future. And with CommonJS supported, there’s no reason not to support CoffeeScript as well—there’s nothing MDG needs to do really to keep the CoffeeScript support that Meteor already has. That’s why this PR is simply documentation, explaining to CoffeeScript users how to use modules, without any new code to fix any incompatibilities between Meteor 1.3 and CoffeeScript.

**Comment by GeoffreyBooth** [comment]
You're not using this documentation?

**Comment by lorensr** [comment]
+1 on reopening. Might be better if the info were placed here:

http://docs.meteor.com/#/full/coffeescript

Also with a link to it from this section: 

http://docs.meteor.com/#/full/modules

**Comment by benjamn** [comment]
Sorry, this got closed only because the `release-1.3` branch was deleted post-launch. Can you resubmit the PR?

**Comment by GeoffreyBooth** [comment]
Resubmitted as #6636.

**Comment by rijk** [comment]
Wow, this was really helpful, thanks @GeoffreyBooth !! +1 for including it in the documentation, had to Google for about half an our before I found this.

**Comment by GeoffreyBooth** [comment]
Thanks. You should encourage @benjamn or @zol to merge it in ;)

**Comment by GeoffreyBooth** [comment]
#6636, I mean.

File: packages/modules/README.md [hunk 1]
```diff
@@ -96,6 +96,44 @@ import {default as Value, a, F} from "./exporter";
...
+can be rewritten to use CommonJS `module.exports`:
+
+```coffeescript
+module.exports.insert = new ValidatedMethod # ...
+module.exports = incompleteCountDenormalizer
+```
+
+You can also simply write `exports` instead of `module.exports` if you prefer.
+
+Note that [backticks](http://coffeescript.org/#embedded) do *[not](https://github.com/meteor/meteor/issues/6000)* work; your `.coffee` files will be transpiled into JavaScript, but not then handed off to the `ecmascript` package for further transpiling. Any JavaScript you type in backticks will be passed through unmodified all the way to the browser or Node.js runtime.
```
Comment (mitar) [inline packages/modules/README.md]: This is not true anymore.

**Pull request closed without merging**